src/CodeViewer/Viewer.cpp

```diff
@@ -329,16 +329,33 @@ void Viewer::DrawSampleCounters(QPaintEvent* event) {
   }
 }
 
+uint64_t Viewer::LargestOccuringLineNumber() const {
+  switch (line_number_types_) {
+    case LineNumberTypes::kNone:
+      return 0;
+    case LineNumberTypes::kOnlyMainContent:
+      return largest_occuring_line_number_main_content_.value_or(blockCount());
+    case LineNumberTypes::kOnlyAnnotatingLines:
+      return largest_occuring_line_number_annotating_lines_.value_or(0);
+    case LineNumberTypes::kBoth:
+      return std::max(largest_occuring_line_number_main_content_.value_or(blockCount()),
```

Comment:
We could in the future modify this in order to only take into account the LineNumbers of the AnnotatingLines, am I wrong?

Reply:
I don't understand. If the line number type is set to `kOnlyAnnotatingLines` only these will be shown and taken into account. Does that make sense?